2016/04/07 19:01:49 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: could not find driver in C:\xampp\htdocs\yii\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\yii\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/siadm/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/siadm/index.php?r=gii/default/index
---
